heidegger_index/migrations/0010_work_complete.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.1 on 2023-01-05 09:46
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('heidegger_index', '0009_lemma_gnd_id'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='work',
+            name='complete',
+            field=models.BooleanField(default=False),
+        ),
+    ]
```

Comment:
Let even op dat er nu twee migration files van 0009 afhankelijk zijn. Ik zou als ik jou was in #25 even de migration files combineren (0009 en 0010 verwijderen en opnieuw een `makemigrations` doen).